chromium/allocator.go

```diff
@@ -172,20 +172,27 @@ readLoop:
 		// Copy the rest of the output in a separate goroutine, as we
 		// need to return with the websocket URL.
 		go func() {
-			io.Copy(forward, bufr)
+			_, _ = io.Copy(forward, bufr)
 			done()
 		}()
 	}
 	return wsURL, nil
 }
 
 // Allocate starts a new local browser process
-func (a *Allocator) Allocate(ctx context.Context, launchOpts *common.LaunchOptions) (*common.BrowserProcess, error) {
+func (a *Allocator) Allocate(ctx context.Context, launchOpts *common.LaunchOptions) (_ *common.BrowserProcess, err error) {
```

Comment:
Should we rename `err` to something else? It's being shadowed several times below.

Reply:
https://github.com/grafana/xk6-browser/pull/211/commits/1f9c569bf54bdcfcff18eec8506c171c576a2411#diff-519450d276eec66ebdf75fca8ffc07bb0179a1b647538b0be9da68d2faf52c39L187-L189

Reply:
`rerr` is never written to now, so it will always be `nil`.

Reply:
It just catches the returning error message, are you sure?

See: https://go.dev/play/p/bRqOrWOnqw2

Reply:
Ah, sure, you're right. :+1: